bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -186,7 +186,10 @@ public synchronized void modified(Map<String, Object> config) {
                 for (InterfaceAddress interfaceAddress : interfaceAddresses) {
                     final InetAddress addr = interfaceAddress.getAddress();
                     if (addr instanceof Inet4Address && !addr.isLinkLocalAddress() && !addr.isLoopbackAddress()) {
-                        broadcastAddresses.add(interfaceAddress.getBroadcast().getHostAddress());
+                        InetAddress broadCast = interfaceAddress.getBroadcast();
```

Comment:
the internet claims that "broadcast" is a single word

Reply:
You believe everything that the internet claim, right? :P Fixed...